daxa/mission/erosita.py

```diff
@@ -514,7 +481,7 @@ def download(self, num_cores: int = NUM_CORES):
                 raised_errors = []
 
                 # This time, as we want to use multiple cores, I also set up a Pool to add download tasks too
-                with tqdm(total=len(fields_to_be_downloaded ), desc="Downloading {} data".format(self._pretty_miss_name)) \
+                with tqdm(total=len(download_links), desc="Downloading {} data".format(self._pretty_miss_name)) \
```

Comment:
When I deleted all of the data I acquired the first time I tested this, and tried to download on multiple cores again, it did do the download but afterwards an index error got thrown.

Reply:
this was an issue with the _directory_formatting function.

When I changed the _download_call to have field specific download folders instead of having them all in temp_download, there was another layer of folders that _directory_formatting couldnt deal with, so I have added in another check for the extra layer.

This is resolved in the commit that mentions _directory_formating